.github/stale.yml

```diff
@@ -1,15 +1,15 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+daysUntilStale: 5
 
 # Number of days of inactivity before a stale issue is closed
-daysUntilClose: 7
+daysUntilClose: 5
```

Comment:
Let's keep at 7d since 5d (with a weekend in between) is too short for the author to respond to a stale tag before getting the issue closed.